src/App.jsx

```diff
@@ -1,17 +1,44 @@
+import React, { useState } from 'react';
 import './App.css';
+import ChatEntry from './components/ChatEntry';
+import ChatLog from './components/ChatLog';
+import chatMessages from './data/messages.json';
 
 const App = () => {
+  // const [entries, setEntries] = useState(chatMessages);
+  const [entries, setEntries] = useState(
+  chatMessages.map((msg) => ({ ...msg, liked: msg.liked ?? false }))
+);
+
+  const toggleLike = (id) => {
+    const updatedEntries = entries.map((entry) => {
+      if (entry.id === id) {
+        return { ...entry, liked: !entry.liked };
+      }
+      return entry;
+    });
+    setEntries(updatedEntries);
+  };
+
+  const totalLikes = entries.filter((entry) => entry.liked).length;
+  const localSender = "Vladimir";
+
   return (
     <div id="App">
       <header>
-        <h1>Application title</h1>
+        <h1>Chat Log</h1>
```

Comment:
An element missing from your project is the header "Chat Between Vladimir and Estragon". Here you have "Chat Log". 

@NataliaRaz How would you create a header that reflects the screenshot below? It can be done by hardcoding the string in the `h1` tag, but I'm wondering how would you programmatically get the names of the participants in the chat?

Please respond to this comment with your answer, thank you!

<img width="851" alt="image" src="https://github.com/user-attachments/assets/9425330e-3a71-4f6f-9471-e198f074805c" />